backend/src/data/entities/RefreshToken.ts

```diff
@@ -10,7 +10,6 @@ export class RefreshToken extends AbstractEntity {
   @Column()
   value: string;
 
-  @OneToOne(() => User)
-  @JoinColumn()
+  @ManyToOne(() => User, user => user.refreshTokens)
```

Comment:
Please take a look at how `@ManyToOne` relations are implemented in other columns and implement it this way. we need to add `@RelationId` decorator